### PR TITLE
Added more configs to dev ispn configuration for console test needs.

### PR DESCRIPTION
**Pull request by andyuk1986**
backport #10999

### CODE REVIEW

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
@tristantarrant what do you think on this one?

**Comment by tristantarrant** [comment]
I'm ok with the metrics. I'm not ok with the default cache and the configuration template. Those do not exist in a real server, and building against them would create a disalignment.

File: server/runtime/src/main/server/server/conf/infinispan-dev-mode.xml [hunk 1]
```diff
@@ -35,10 +35,15 @@
    </jgroups>
 
    <cache-container name="default" statistics="true">
+      <metrics accurate-size="true"/>
       <transport cluster="${infinispan.cluster.name:cluster}" stack="${infinispan.cluster.stack:tcp}" node-name="${infinispan.node.name:}"/>
       <security>
          <authorization/>
       </security>
+      <local-cache name="default"/>
```
Comment (tristantarrant) [inline server/runtime/src/main/server/server/conf/infinispan-dev-mode.xml:43]: Replace with a distributed cache that inherits from the template defined below.

Comment (karesti) [reply server/runtime/src/main/server/server/conf/infinispan-dev-mode.xml:43]: I pushed a change

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**